### PR TITLE
marathon-lb: Bump to 1.7.0

### PR DESCRIPTION
**Pull request by nlsun**

### CODE REVIEW

**Comment by nlsun** [comment]
@ryadav88 ready for review

**Comment by nlsun** [comment]
@kensipe ready for review

File: repo/packages/M/marathon-lb/19/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "2.0",
```
Comment (kensipe) [inline repo/packages/M/marathon-lb/19/package.json]: I'm not sure why this is still version 2.0.  this is very old.  version 3.0 is the latest and we are moving to 4 soon.

File: repo/packages/M/marathon-lb/19/package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "name": "marathon-lb",
+  "version": "1.7.0",
+  "scm": "https://github.com/mesosphere/marathon-lb",
+  "description": "HAProxy configured using Marathon state",
+  "maintainer": "support@mesosphere.io",
+  "tags": ["loadbalancer", "service-discovery", "reverse-proxy", "proxy", "haproxy"],
+  "preInstallNotes": "We recommend at least 2 CPUs and 1GiB of RAM for each Marathon-LB instance. \n\n*NOTE*: ```Enterprise Edition``` DC/OS requires setting up the Service Account in all security modes. \nFollow these instructions to setup a Service Account: https://docs.mesosphere.com/administration/id-and-access-mgt/service-auth/mlb-auth/",
```
Comment (kensipe) [inline repo/packages/M/marathon-lb/19/package.json]: I understand you are making a simple version update.   Making a minor non-blocking comment here.   It probably makes sense to mention in the pre install notes that the 2 CPU, 1G recommendation is on the public node and that a public node must be available by default. (but perhaps no one reads these pre-install notes:)

**Comment by nlsun** [comment]
@kensipe Actually there is a diff, I separated it out into 2 commits.

File: repo/packages/M/marathon-lb/19/package.json (hunk 1 above)
Comment (nlsun) [inline repo/packages/M/marathon-lb/19/package.json]: That's true, we can address this in a separate patch

File: repo/packages/M/marathon-lb/19/package.json (hunk 2 above)
Comment (nlsun) [inline repo/packages/M/marathon-lb/19/package.json]: That's true, we can address that in a separate patch

**Comment by nlsun** [comment]
@ryadav88 We can address those in a separate patch.

**Pull request merged**